.github/workflows/test_dbt_models.yaml

```diff
@@ -52,7 +52,7 @@ jobs:
 
       - name: Extract test results
         run: |
-          python3 ../.github/scripts/transform_dbt_test_results.py \
+          python3 scripts/transform_dbt_test_results.py \
```

Comment:
While the `pip install` command above provides the `dbt/` subdir as a prefix to this command, we don't need to do that here since the `working-directory` is already set to `dbt` on line 57 below.